### PR TITLE
Run the ruby and sass linters when the default rake task is run

### PR DESCRIPTION
**Pull request by AndrewVos**
This should help us stop committing code that will fail the linter under CI.

### CODE REVIEW

**Comment by AndrewVos** [comment]
It just occurred to me that this will run the linters twice, because in jenkins.sh `rake` gets executed.

Is there a reason why linting does not happen on master?

**Comment by jamiecobbett** [comment]
I think your commit should say "run ruby and sass linters as part of the default rake task". Additionally, does this overwrite the default rake task that runs all tests?

**Comment by AndrewVos** [comment]
@jamiecobbett It doesn't appear so, but I'll switch to another way so it's more explicit.

```
[~/.../government-frontend] run-linter-by-default ∞ rake
govuk-lint-ruby --diff --format clang Gemfile app test config
warning: parser/current is loading parser/ruby21, which recognizes
warning: 2.1.8-compliant syntax, but you are running 2.1.4.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.

0 files inspected, no offenses detected
govuk-lint-sass app
Run options: --seed 8519

# Running:

....................................................................................................

Finished in 0.890031s, 112.3557 runs/s, 212.3523 assertions/s.

100 runs, 189 assertions, 0 failures, 0 errors, 0 skips

```

File: lib/tasks/lint.rake [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+task :lint do
+  if ENV["JENKINS"]
+    sh "govuk-lint-ruby --format html --out rubocop-${GIT_COMMIT}.html --format clang Gemfile app test config"
+  else
+    sh "govuk-lint-ruby --format clang Gemfile app test config"
+  end
+  sh "govuk-lint-sass app"
+end
+
+Rake::Task[:default].enhance [:lint]
```
Comment (AndrewVos) [inline lib/tasks/lint.rake]: @jamiecobbett more explicit form ^

Comment (boffbowsh) [inline lib/tasks/lint.rake]: Are both lines 2 and 12 needed?

Comment (AndrewVos) [inline lib/tasks/lint.rake]: @boffbowsh Nope! Can't believe I missed that! Thanks.

**Pull request merged**